WordPress/gutenberg
# Fix: Change dropdown button label to 'screen options.'

## What?
Fixes: #63029 

## Testing Instructions

- Go to the Site Editor > Pages.
- At the top of the pages list, hover or focus the View Options icon button.
- Observe the button label  should be... Screen options.
- test with languages other than English, Spanish or Italian and please report in the comments below all the incorrect translations you may find.
